ex1.rb

```diff
@@ -0,0 +1,46 @@
+# count sum to 100
+
+sum = 0;
+for i in 1..100
+  sum += i
+end
+
+puts sum
+
+# print 1 to 20 divisible by 3
+
+puts "Numbers divisible by 3: "
+for i in 1..20
+  if i % 3 == 0
```

Comment:
viết như này gọn hơn k e?
```suggestion
    puts i if i % 3 == 0
```

Reply:
em xin tiếp thu ạ